### ISSUE TITLE
Try some more complex models

### ISSUE THREAD

**Issue by SCAZT** [COrtsJosep/landscape-aesthetics#52]
Try to explore if there is other complex model that have better performances. 

- [x]  Give weights to different labels to decrease the impact of the data imbalance.
- [x]  Model construction
- [x]  Training
- [x]  Evaluation

**Comment by SCAZT** [comment]
The model is facing an overfitting issue. Solutions as follows:

- [ ]  adjust learning rate
- [ ]  add an regulation term/add dropout layer
- [ ]  Early stopping(maybe?)
